go.sum

```diff
@@ -1,6 +1,8 @@
 github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c=
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
+github.com/getstream/easyjson v0.0.0-20190923162548-5adc7b1f33fa h1:Yw0xAzmfvWTFyjnsO0r8Rfx8BBqU6FDnF/WfNk0qgtg=
```

Comment:
this probably should be rolled back?

Reply:
Good catch. It kept trying to import itself while I was fixing the name, thought I got all of them.